lib/ConsumerImpl.cc

```diff
@@ -1220,7 +1287,7 @@ void ConsumerImpl::redeliverMessages(const std::set<MessageId>& messageIds) {
             LOG_DEBUG("Sending RedeliverUnacknowledgedMessages command for Consumer - " << getConsumerId());
         }
     } else {
-        LOG_DEBUG("Connection not ready for Consumer - " << getConsumerId());
+        LOG_WARN("Connection not ready for Consumer - " << getConsumerId());
```

Comment:
Please revert this unrelated change. It should not be a warning log when a `shared_ptr` is expired.